packages/arb-token-bridge-ui/src/hooks/useNetworkTPS.ts

```diff
@@ -0,0 +1,53 @@
+import axios from 'axios'
+import { load } from 'cheerio'
+import { getExplorerUrl, isNetwork } from 'src/util/networks'
+import useSWR from 'swr'
+import { useNetworksAndSigners } from './useNetworksAndSigners'
+
+const emptyStats = { tps: null }
+
+const fetchNetworkTPS = async (l2ChainId: number) => {
+  // currently we only support TPS information for Arb-one and nova
+  // return null for rest of the networks
+  const canFetchTPS =
+    isNetwork(Number(l2ChainId)).isArbitrumNova ||
+    isNetwork(Number(l2ChainId)).isArbitrumOne
+  if (!canFetchTPS) return emptyStats
+
+  // url from where we'll fetch stats
+  const explorerUrl = getExplorerUrl(Number(l2ChainId))
+
+  // for 403 or CORS blocked errors while scraping external endpoints, we use cors-proxy
+  const finalUrl = `https://corsproxy.io/?${encodeURIComponent(explorerUrl!)}`
```

Comment:
Using corsproxy.io bypasses any CORS issues arising because it uses a proxy server to access the external website.